src/eigen-qld/QLD.h

```diff
@@ -92,9 +96,12 @@ class QLD
    * example we consider a matrix \a B that is the block of another one \a M, the
    * leading dimension of \a B will be the row dimension of \a M.
    */
-  EIGEN_QLD_API void problem(int nrvar, int nreq, int nrineq, int ldq = -1);
+  EIGEN_QLD_API void problem(int nrvar, int nreq, int nrineq, int ldq = -1, int lda = -1);
 
-  /** Return the result from the latest call to QLD::solve. */
+  /** Same as QLD::problem, but avoid uncessary matrix resizes. */
+  EIGEN_QLD_API void problemNoOverhead(int nrvar, int nreq, int nrineq, int ldq = -1, int lda = -1);
+
+  /** Return the result from the latest call to QLD::solve or QLD::solveNoOverhead. */
```

Comment:
Should probably indicate that you cannot call `::solve` if you only initialized the problem like this